README.md

```diff
@@ -14,26 +17,28 @@ Feel free to add other follow worthy Twitter accounts.
 
 # People
 
-* [@chriseidhof](https://twitter.com/chriseidhof) - creator of Objc.io.
-* [@jesse_squires](https://twitter.com/jesse_squires) - curates the Swift Weekly Brief.
-* [@krausefx](https://twitter.com/krausefx) - creator of fastlane.
-* [@natashatherobot](https://twitter.com/natashatherobot) - trySwift organizer and writes the Swift news newsletter.
-* [@olebegemann](https://twitter.com/olebegemann) - co-authored Advanced Swift and blogging on oleb.net.
-* [@orta](https://twitter.com/orta) - helped build CocoaPods and Danger.
-* [@steipete](https://twitter.com/steipete) - creator of PSPDFKit.
-* [@stroughtonsmith](https://twitter.com/stroughtonsmith) - iOS developer, discovering details about future releases.
+* [@chriseidhof](https://twitter.com/chriseidhof) - creator of Objc.io
+* [@krausefx](https://twitter.com/krausefx) - creator of fastlane
```

Comment:
You accidentally removed @jesse_squires . Please readd the handle and then it's ready to merge 👍